ssl/ssl_versions.cc

```diff
@@ -335,44 +335,54 @@ BSSL_NAMESPACE_END
 using namespace bssl;
 
 int SSL_CTX_set_min_proto_version(SSL_CTX *ctx, uint16_t version) {
+  ctx->conf_min_version_use_default = version == 0;
```

Comment:
NP: This (and the others like it) would be easier to visually parse as:
```
ctx->conf_min_version_use_default = (version == 0);
```

Reply:
sure, i'll do that.